packages/mcdoc/src/runtime/attribute/builtin.ts

```diff
@@ -30,6 +30,19 @@ const idValidator = validator.alternatives<IdConfig>(
 	() => ({}),
 )
 
+interface RegexConfig {
+	pattern: string
+	message?: string
+}
+
+const regexMatchValidator = validator.alternatives<RegexConfig>(
```

Comment:
Should probably be named `matchRegexValidator` and `MatchRegexConfig` for consistency